integration-tests/src/test/java/com/datastax/oss/driver/api/core/metadata/NodeStateIT.java

```diff
@@ -119,6 +144,7 @@ public void should_keep_regular_node_up_when_still_one_connection() {
         .as("Reconnection started")
         .before(10, TimeUnit.SECONDS)
         .becomesTrue();
+    inOrder.verify(nodeStateListener, never()).onDown(metadataRegularNode);
```

Comment:
Awesome 👍 .  Like that could use existing tests to validate the state listener behavior.